### DISCUSSION TITLE
Can you delete RGraph version 606 (no full stop in the version number) please?

### DISCUSSION THREAD

**Discussion by heyesr** [cdnjs/cdnjs#14156] [General]
Hi,

Normally I release RGraph versions with version numbers like this: 6.08 6.07 6.06 6.05 etc, But I inadvertently released one with a version number of 606 (without the full-stop). I've deleted the Git branch and tag - is it possible to get this removed from CDNjs too?

Thanks!

Richard

**Comment by MattIPv4** [comment]
👋 It is not possible for us to remove versions that are published on the CDN, as users may be reliant on them.

**Comment by heyesr** [comment]
Hi,

Ok - is it possible to stop it from showing up first in the dropdown?

Cheers.

**Reply by MattIPv4** [reply]
Please can you open an issue over on cdnjs/packages to track this request. It's not something we can currently do, but it has been raised before (https://github.com/cdnjs/packages/issues/926), so also tracking the ask for it with this library would be great :)

**Reply by heyesr** [reply]
Ok thanks - done that: https://github.com/cdnjs/packages/issues/1307
